docs/practices/workflows/gas_estimations.md

```diff
@@ -48,13 +48,13 @@ cargo run --release -p runtime-params-estimator --features required -- \
 ```
 
 You might also want to run a hardware-agnostic estimation using the following
-command. It uses `docker` and `qemu` under the hood, so it will be quite a bit
-slower. You will need to install `docker` to run this command.
+command. It uses `podman` and `qemu` under the hood, so it will be quite a bit
+slower. You will need to install `podman` to run this command.
 
 ```bash
 cargo run --release -p runtime-params-estimator --features required -- \
     --accounts-num 20000 --additional-accounts-num 2000000 \
-    --iters 3 --warmup-iters 1 --metric icount --docker \
+    --iters 3 --warmup-iters 1 --metric icount --containerize \
```

Comment:
Hm, It fails with `--containerize` on my machine:

```bash
     Running `target/release/runtime-params-estimator --accounts-num 20000 --additional-accounts-num 2000000 --iters 3 --warmup-iters 1 --metric icount --containerize --costs=action-function-call-base-send-not-sir,action-function-call-base-send-sir,action-function-call-base-exec`
error: unexpected argument '--containerize' found

Usage: runtime-params-estimator --accounts-num <ACCOUNTS_NUM> --additional-accounts-num <ADDITIONAL_ACCOUNTS_NUM> --iters <ITERS> --warmup-iters <WARMUP_ITERS> --metric <METRIC>
```

Reply:
@MCJOHN974: the command seems to work for me.  Are you sure that your repo is up-to-date with the `master` branch?

Reply:
Yes, I runned it on last commit of `master`. But after updating `podman` I got a new error message:
```bash
Running `target/release/runtime-params-estimator --accounts-num 20000 --additional-accounts-num 2000000 --iters 3 --warmup-iters 1 --metric icount --containerize --costs=action-function-call-base-send-not-sir,action-function-call-base-send-sir,action-function-call-base-exec`
[elapsed 00:11:43 remaining 00:00:00] Writing into storage ████████████████████ 2000000/2000000
Error: failed to run `podman images -q rust-emu:rust-1.76.0`
```

Reply:
Depending on your operating system you _may_ need something like [this](https://www.mankier.com/5/containers-policy.json#Examples-Completely_disable_security,_allow_all_images,_do_not_trust_any_signatures) in `~/.config/containers/policy.json` as well.

Reply:
looks like `podman` had to be updated.  Viktar is doing that now.